### PR TITLE
blog

### PR DESCRIPTION
**Pull request by kaushikichhabra**
url: https://fossasiasblog.wordpress.com
title FOSS ASIA AND ME 
img: IMG_4670
author_name: Kaushiki Chhabra

### CODE REVIEW

**Comment by kaushikichhabra** [comment]
url: https://fossasiasblog.wordpress.com
title FOSS ASIA AND ME 
img: IMG_4670
author_name: Kaushiki Chhabra

File: _data/blogs.yml [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+# - url: URL to the blog
+#   title: Title of the blog
+#   img: Image of the blog
+#   author_name: Name of the author of the blog (most likely you)
+
+- url: https://participationgcin.wordpress.com/
+  title: GET LOST IN THE DEPTH OF LEARNING WITH GOOGLE CODE-IN
+  img: sudhanshuedoc.jpg
+  author_name: Sudhanshu Ranjan
+
```
Comment (kaushikichhabra) [inline _data/blogs.yml:15]: url: https://fossasiasblog.wordpress.com
title FOSS ASIA AND ME 
img: IMG_4670
author_name: Kaushiki Chhabra

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Please close this PR, and make a new one!
Or,
Remove conflicts, rebase, squash commits :+1:

**Review by akshatnitd — COMMENTED** [review]
@kaushikichhabra I would suggest you to open up a new PR for this! :)

**Review by nikit19 — COMMENTED** [review]
Please open a new pull request and send it from your forked repository

**Pull request closed without merging**